packages/rspack-test-tools/tests/legacy-test/Output.test.js

```diff
@@ -52,7 +52,7 @@ describe("Output", () => {
 	});
 
 	it("should be cleared the build directory", done => {
-		const outputDist = "dist/output";
+		const outputDist = path.resolve(__dirname, "../js/legacyTest");
```

Comment:
should use `js/legacy-test/output-{case-name}` to prevent conflicting?

Reply:
Thanks. It's fixed now.